crates/kitsune_p2p/dht/src/arq.rs

```diff
@@ -580,7 +581,7 @@ pub fn power_and_count_from_length(dim: impl SpaceDim, len: u64, max_chunks: u32
     let dim = dim.get();
     assert!(len <= U32_LEN);
     let mut power = 0;
-    let mut count = (len / dim.quantum as u64) as f64;
+    let mut count = (len as f64 / dim.quantum as f64).ceil();
```

Comment:
Division was rounding down here which always resulted in an arq that was up to 1 quantum smaller than the arc in standard coordinates. This had strange behaviour for small standard arcs (less than the quantum size) where the arc would be mapped to empty rather than the quantum size. That seems wrong to me, the arq does not represent the arc if it loses part of the arc space. Especially for a small arc, it seems wrong for a small arc to become empty when it wasn't empty. Empty is really a special value.

Reply:
Okay, that sounds like something that'd be better off on its own PR

Reply:
Moved to an issue https://github.com/holochain/holochain/issues/3976